var/spack/repos/builtin/packages/dia/package.py

```diff
@@ -34,15 +34,18 @@ class Dia(Package):
 
     depends_on('intltool', type='build')
     depends_on('gtkplus@2.6.0:')
-    depends_on('cairo')
-    depends_on('libpng')
```

Comment:
To be sure, you can check in the build logs of `dia` if `cairo` or `libpng` are mentioned anyhow.